dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import {
+  InputGroup,
+  InputGroupText,
+  DatePicker,
+  isValidDate,
+  Button,
+} from "@patternfly/react-core";
+import "./index.css";
+import { filterData } from "utils/filterDataset";
+import {dateFromUTCString } from "utils/constructDate";
+import {bumpToDate} from "utils/bumpDate";
+import { getTodayMidnightUTCDate } from "utils/getMidnightUTCDate";
```

Comment:
It looks like you need to run Prettier over your code again (i.e., lines 11 & 12 have different formatting inside the braces from lines 10 & 13...). 🙂